scripts/seed/reset-maker.ts

```diff
@@ -22,13 +22,17 @@ export async function resetMaker(actor: Actor) {
 	const dsr = xToRontox(1n)
 	if (await pot.dsr_() !== dsr) await setDsr.setDsr(dsr)
 
-	// set ETH price
-	await vat.file(ethId, stringLiteralToBigint('spot'), xToRontox(190n))
+	// set ETH launch parameters (pulled from official deployed contracts)
+	await vat.file(ethId, stringLiteralToBigint('rate'), 1000389702408434119151108112n)
+	await vat.file(ethId, stringLiteralToBigint('spot'), 122593498983333333333333333333n)
+	await vat.file(ethId, stringLiteralToBigint('line'), 0n)
+	await vat.file(ethId, stringLiteralToBigint('dust'), 20000000000000000000000000000000000000000000000n)
 
-	// set ETH debt ceiling
-	const attorontodaiDebtCeiling = xToAttorontox(100_000_000n)
+	// set ETH debt ceiling (from official launch trigger)
+	const attorontodaiDebtCeiling = xToAttorontox(153_000_000n)
+	const ethCollateralAttorontodaiDebtCeiling = xToAttorontox(50_000_000n)
 	await vat.file2(stringLiteralToBigint('Line'), attorontodaiDebtCeiling)
-	await vat.file(ethId, stringLiteralToBigint('line'), attorontodaiDebtCeiling)
+	await vat.file(ethId, stringLiteralToBigint('line'), ethCollateralAttorontodaiDebtCeiling)
```

Comment:
Updated one of the scripts with Maker launch parameters so we can test a little closer to the real deal.  We may want to bump up our test account ETH amounts to the hundreds of millions so we can exercise behavior near the cap...

Reply:
0x913 still has ungodly amounts of ETH? or put them on the first 10 HD?